_opensearch/snapshots/sm-dashboards.md

```diff
@@ -95,3 +95,57 @@ Use the steps below to take a snapshot manually. If you need to restore a snapsh
     - **Ignore unavailable indices**
     - **Allow partial snapshots**
 1. Select the **Add** button.
+
+## Restore a snapshot
+
+1. On the top menu bar, go to **OpenSearch Plugins > Snapshot Management**.
+1. In the left panel, under **Snapshot Management**, select **Snapshots**. The **Snapshots** tab is selected by default.
+1. Select the checkbox next to the snapshot you want to restore: 
+    <img src="{{site.url}}{{site.baseurl}}/images/restore-snapshot/restore-snapshot-main.png" alt="Snapshots">{: .img-fluid}
+
+    You can only restore one snapshot at a time.
+    {: .note}
+    You can only restore snapshots with the status of `Success` or `Partial`. The status of the snapshot is displayed in the **Snapshot status** column.
```

Comment:
To be clear, there can be multiple snapshot processes initiated one after another, but only the most recent one's progress can be viewed.